services-turf/src/test/java/com/mapbox/turf/TurfMiscTest.java

```diff
@@ -340,4 +346,53 @@ public void testTurfPointOnLinePointsOnSidesOfLines() throws TurfException {
       assertNotEquals(snapped, last);
     }
   }
+
```

Comment:
Are these test not relevant on the Java side or am I overlooking them? https://github.com/Turfjs/turf/blob/master/packages/turf-line-slice-along/test.js#L11-L39